### PR TITLE
Create plan on team create

### PR DESCRIPTION
**Pull request by usirin**
Right now team plan is not created at team creation, this PR makes sure that team has a plan if the config is not `default`
## How Has This Been Tested?

Manually and with unit tests
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

/cc @gokmen @sinan @szkl

### CODE REVIEW

File: servers/lib/server/handlers/createteam.coffee [hunk 1]
```diff
@@ -224,10 +225,18 @@ afterGroupCreateKallback = (res, params) ->
...
       # add other parallel operations here
-      (fin) -> createInvitations client, invitees, (err) ->
+      (fin) ->
+        createInvitations client, invitees, (err) ->
           console.error 'Err while creating invitations', err  if err
           fin()
 
+      (fin) ->
+        params = { sessionToken: client.sessionToken }
+        params.coupon = coupon  if coupon
```
Comment (cihangir) [inline servers/lib/server/handlers/createteam.coffee]: thanks for adding coupon support 👍

File: servers/lib/server/handlers/createteam.test.coffee [hunk 2]
```diff
@@ -14,6 +14,7 @@ KONFIG                              = require 'koding-config-manager'
 { testCsrfToken }                   = require '../../../testhelper/handler'
 { generateRegisterRequestParams }   = require '../../../testhelper/handler/registerhelper'
 { generateCreateTeamRequestParams } = require '../../../testhelper/handler/teamhelper'
+{ Status }                          = require '../../../../client/app/lib/redux/modules/payment/constants'
```
Comment (cihangir) [inline servers/lib/server/handlers/createteam.test.coffee]: woo–hoo using client side code on backend :D 

I dont know if it is good or bad tho in general 😅

Comment (cihangir) [inline servers/lib/server/handlers/createteam.test.coffee]: @sinan @gokmen wdtyt?

Comment (sinan) [inline servers/lib/server/handlers/createteam.test.coffee]: they're just `constants` i don't see why not.

Comment (sinan) [inline servers/lib/server/handlers/createteam.test.coffee]: only problem would be that if we split the codebase in the future we'd need to duplicate them, which is also fine imo.

Comment (gokmen) [inline servers/lib/server/handlers/createteam.test.coffee]: agree with @sinan 👍

**Pull request merged**